### DISCUSSION TITLE
About Turborepo .cache folder creation in ./node_modules directory.

### DISCUSSION THREAD

**Discussion by manippoudel** [vercel/turborepo#1855] [Help] [answered]
I'm new to turborepo and trying to run it on my local computer. Even after deleting the **.cache** directory from node modules, I see the **.cache** directory being created by the command `turbo run`. Can anyone suggest a way to disable the creation of **.cache** folder when starting the turbo using `turbo run`?

**Comment by mehulkar** [comment, marked as answer]
- You can [change the caching directory](https://turbo.build/repo/docs/reference/command-line-reference#--cache-dir) with `turbo run <task> --cache-dir`
- Or you can disable caching altogether
	- with [`--no-cache` from the CLI](https://turbo.build/repo/docs/reference/command-line-reference#--no-cache) (affects any task you run)
	- with [`cache:false`](https://turbo.build/repo/docs/reference/configuration#cache) in the turbo.json config
